<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSettingMachineForm</class>
 <widget class="QWidget" name="QSettingMachineForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_settingmachine_xishu">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settingmachine_jiandna">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settingmachine_lengningshui">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settingmachine_xiangxi">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settingmachine_fangfa">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_settingmachine_zijian">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settingmachine_xishu">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_setttingmatchine_save">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_settingmachine_fangfaxishu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_setttingmatchine_fangfaxishu">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial_year">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial_day">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>31</number>
   </property>
  </widget>
  <widget class="QDial" name="dial_month">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QDial" name="dial_hour">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial_second_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial_muit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_year">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>2015</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_mouth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_day">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hour">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>23</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_muit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>59</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_second">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>59</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xishu1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dayin">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zijian">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xishu2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btnJiandan">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btnComplex">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
